.travis.yml

```diff
@@ -1,23 +1,37 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
```

Comment:
PHP 5.6 still needs to be listed here, so that it runs with and without `--prefer-lowest`.